helm-to-operator-codegen-sdk/config/enum_module_mapping.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Are these exhaustive lists? Would it be better to fetch this data from somewhere official?

Reply:
I have a go-script that fetches the "types.go" of the package and then, It generates both the json files using appropriate regex-pattern-matching,
Further, I have seen that the golang documentation also has the same information about the types in package (Eg: https://pkg.go.dev/k8s.io/api@v0.27.3/core/v1). I will check if there's an official api that I can call or (maybe we can web-scrape the same values).